src/modules/75-cf/strings/strings.en.yaml

```
@@ -165,14 +165,13 @@ featureFlags:
   archiving:
     archiveFlag: Archive Flag
     archiveSuccess: Flag successfully archived
-    cannotArchive: '{{ flagName }} cannot be archived because it is still being used by the following flags as a prerequisite'
-    confirmFlag: Type the flag name to confirm
-    mismatchNameError: The flag name does not match
+    cannotArchive: '{{ flagId }} cannot be archived because it is still being used by the following flags as a prerequisite'
+    confirmFlag: Type the flag Id to confirm
+    mismatchIdError: The flag Id does not match
```

Try to stick to `identifier` to match the wording of the rest of Harness.
```suggestion
    confirmFlag: Type the flag identifier to confirm
    mismatchIdError: The flag identifier does not match
```